book/src/concepts.md

```diff
@@ -218,7 +218,7 @@ Ok so here's what goes through cargo-dist's brains when you run it:
 1. Read in the workspace/config/cli-flags
 2. Determine the Announcement Tag (select the Apps) ("v1.0.0")
 3. Determine what Targets we're building for
-3. Call the specific Version of each App a "Release" ("my-app-v1.0.0" or "my-app/v1.0.0")
```

Comment:
Was undoing this part intentional?

Reply:
Yes, this is talking about how cargo-dist generates artifact names, and it doesn't generate `/` in a file name.